code/game/objects/items/surgery_tray.dm

```diff
@@ -15,26 +15,33 @@
 
 	/// If true we're currently portable
 	var/is_portable = TRUE
-	/// List of things that we spawn containing
-	var/list/initial_contents = list(
-		/obj/item/blood_filter,
-		/obj/item/bonesetter,
-		/obj/item/cautery,
-		/obj/item/circular_saw,
-		/obj/item/clothing/mask/surgical,
-		/obj/item/hemostat,
-		/obj/item/razor/surgery,
-		/obj/item/retractor,
-		/obj/item/scalpel,
-		/obj/item/stack/medical/bone_gel,
-		/obj/item/stack/sticky_tape/surgical,
-		/obj/item/surgical_drapes,
-		/obj/item/surgicaldrill,
-	)
 
 /obj/item/surgery_tray/deployed
 	is_portable = FALSE
 
+/obj/item/surgery_tray/full
+
+/obj/item/surgery_tray/full/deployed
+	is_portable = FALSE
+
+/obj/item/surgery_tray/proc/populate_contents()
+
+/obj/item/surgery_tray/full/populate_contents()
```

Comment:
This is in an odd order.
Please move it so that we:

- Define surgery tray
- Define all of its procs
- Define subtypes
- Implement overrides of each subtype before describing the next subtype

And while we're at it make it
```
/// Fills the tray with items it should contain on creatuib
/obj/item/surgery_tray/proc/populate_contents()
	return
```
Just for neatness

Reply:
Done!